recipes/r-rmvp/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set version = '0.99.15' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmvp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rMVP_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rMVP/rMVP_{{ version }}.tar.gz
+  sha256: e6b0d664be5ba7a1a1a7dccdd835183e4c867baef0a316f6e9c0ca0ec696ef56
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-bh
+    - r-mass
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-bigmemory
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-bh
+    - r-mass
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rcppeigen
+    - r-rcppprogress
+    - r-bigmemory
+
+test:
+  commands:
+    - $R -e "library('rMVP')"           # [not win]
+    - "\"%R%\" -e \"library('rMVP')\""  # [win]
+
+about:
+  home: https://github.com/XiaoleiLiuBio/rMVP
+  license: Apache License 2.0
```

```suggestion
  license: Apache-2.0
```

Apache-2.0 is the spdx short identifier, see https://spdx.org/licenses/Apache-2.0.html